src/main/shadow/cljs/devtools/server/prepl.clj

```diff
@@ -182,7 +182,6 @@
     (<!! thread)))
 
 (comment
-  (require '[shadow.cljs.devtools.api :as shadow])
```

Comment:
Please leave these. They are for quickly testing stuff in the REPL.

Reply:
I removed it because it was being brought in via the ns already. Let me know given this info if you'd still like it to stay.